apps/site-app/src/containers/MainNavigation/MainNavigation.stories.tsx

```diff
@@ -1,19 +1,41 @@
 import React from 'react';
 import { Meta } from '@storybook/react';
 import { MainNavigation } from '.';
-import { withAccounts, withLanguageProvider, withMediaContext } from '@thecointech/storybookutils';
+import { withAccounts, withLanguageProvider, withMediaContext, withReducer } from '@thecointech/storybookutils';
+import { SidebarItemsReducer } from '@thecointech/shared/containers/PageSidebar/reducer';
+import styles from '../App/styles.module.less';
 
 export default {
-  title: 'App/Header',
+  title: 'App/Navigation',
   component: MainNavigation,
   decorators: [
+    withReducer(SidebarItemsReducer),
     withMediaContext,
     withLanguageProvider,
-    withAccounts({
-      active: null,
-      map: {}
-    }),
+    withAccounts(),
   ]
 } as Meta;
 
-export const Header = () => <MainNavigation />;
+SidebarItemsReducer.initialize({
+  items: {
+    header: null,
+    links: [
+      {
+        "name": { "id": 2, "defaultMessage": "Home" },
+        "to": "/",
+        "icon": "home"
+      },
+      {
+        "name": { "id": 3, "defaultMessage": "Top up Balance" },
+        "to": "/balance",
+        "icon": "arrow circle up"
+      },
+    ]
+  }
+})
```

Comment:
I couldn't figure out how to bring in the apps sidebar links here - some combo of storybook/intljs/babel caused a compiler error.